Automattic/woocommerce-payments
# Fix incorrect channel value in transaction description screen for Tap to Pay for Android transactions

Fixes #6735

#### Changes proposed in this Pull Request
Fixes incorrectly displayed channel value for Tap to Pay for Android transactions on transaction details screen.

<img width="631" alt="Screenshot 2023-07-11 at 19 05 03" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/a55b80f3-260d-46d0-ab81-63d63da3ce30">

#### Testing instructions
- Create a Tap to Pay order on Android device
- Navigate to `Payments` -> `Transactions`
- Click on the transaction to be redirected to the transaction details page
- Verify the channel column value is Tap to Pay on Android

-------------------


**Post merge**
